platform/kube/inject/inject.go

```diff
@@ -152,19 +186,34 @@ func injectIntoPodTemplateSpec(p *Params, t *v1.PodTemplateSpec) error {
 	if p.Verbosity > 0 {
 		args = append(args, "-v", strconv.Itoa(p.Verbosity))
 	}
-	if p.MeshConfigMapName != "" {
-		args = append(args, "--meshConfig", p.MeshConfigMapName)
-	}
 
-	ports, err := healthPorts(t)
-	if err != nil {
-		return err
-	}
-	for _, port := range ports {
-		args = append(args, "--passthrough", strconv.Itoa(port))
-	}
+	_, _ = healthPorts(t)
+	/*
+		if err != nil {
```

Comment:
Please add a TODO, that we are effectively losing health checks in K8S and we need to surface this in service model.